mitodl/mitxpro
# Allow only positive values on price and course count External Course/Program

#### Pre-Flight checklist



#### What are the relevant tickets?
None

#### What's this PR do?
- It adds constraints on `Course Count` to be positive on `External Programs`
- Adds constraint on price to be only positive on `External Course and External Program`

#### How should this be manually tested?
- goto `/cms`
- Add an External Program and try to add negative value on `Course Count` field (It should not be allowed now)
- Add any External Program/Course and try to add negative value for `Price` field (It should show validation error message)
 
#### Any background context you want to provide?
While working around some other issue i noticed that negative values for `Course Count` and `Price` were applicable while adding External Courses/Programs. I just created this PR to stop allowing negative values for the fields that should have only positive values
